data/brands/shop/electrical.json

```diff
@@ -4,6 +4,16 @@
     "exclude": {"generic": ["^electrical$"]}
   },
   "items": [
+     {
+      "displayName": "Life Saver",
+      "locationSet": {"include": ["us","ca"]},
+      "tags": {
+        "brand": "Life Saver",
+        "brand:wikidata": "Q122257514",
+        "name": "Life Saver",
+        "shop": "electrical"
```

Comment:
Hm. I'm not sure `shop=electrical` is the best tag here. `shop=car_repair` with some value of `service=` would fit better to me, but I'm not sure. Of already-mapped locations, one is tagged `shop=car_parts` and the rest have no clear specific tag.